content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+
+There are several things staff engineers should do to improve communication in their teams and company:
+* Listen carefully to others and understand their problems and point of view.
+* Make sure the team reaches a collective understanding.
+* [Start with why](https://simonsinek.com/books/start-with-why/) and explain why we work on certain things.
+* Lead by example and improve their communication skills. Show more junior engineers this is often a key to success.
+
+## Bring calm
+![](./calm.jpg)
+
+There are going to be some stressful situations at work. It can be either a tight deadline or an application incident. Staff engineers should
+show their experience and stay calm during the crisis. They should take a lead, communicate clearly and help with prioritization. Their role
+is to help team members do their work better.
+
+Another way to bring calm to the organization is by preparing for different failure scenarios much earlier. Staff engineers should assess
+possible risks even before the project starts. Then, during the ideation or implementation phase, they should systematically review the risks,
+think if mitigation plans still work, and adjust. The best way to get out of the crisis is to foresee it and prevent it from happening.
+
+So what to do to bring calm to the team?
+* Do a risk assessment, and pre-mortems and anticipate failure scenarios before they happen.
+* In critical situations take a lead and help the team solve the problem.
+* And again, communicate clearly and bring clarity.
+
+## Build trust and relationships
+![](./relationship.png)
+
+Last but not least, staff engineers are expected to build and cherish relationships at work. They should have relationships not only with 
+their closest team, but different stakeholders and other colleagues in the organization. They should listen carefully and understand the 
+needs of people and teams. This can help them better identify problems and empower others.
+
+Moreover, it's expected that staff engineers will build credibility and visibility. It's important, that other people and leaders will trust
+they can do a good job. Leadership positions usually bring more "freedom" on what to work, often behind scenes, so it's crucial to keep
+stakeholders up to date on what's going on. Also, when people trust you, they're going to be more open and you're going to gain more from
+the relationship.
+
+Some ideas on how to build trust and relationships:
+* Have regular 1:1s with your team members and other people in the organization.
+* Approach with empathy, listen carefully, and try to understand different points of view.
+* Build your credibility and visibility in the organization, so people know they can rely on you.
+* Be accountable, and communicate clearly what you're going to do and what's not going to happen.
+
+# Summary
+To recap, we went through 5 expectations towards staff engineers. The list is non-exhaustive but contains skills and traits we consider 
```

Comment:
Let's name the five instead. Nobody cares how many of them we listed from within the summary, what they care about is for them get reminded about the early on-the-list ones, that they might have already forgotten about. :D